API-Server/src/main/java/swm/hkcc/LGTM/app/modules/auth/service/AuthServiceImpl.java

```diff
@@ -44,20 +44,25 @@ public class AuthServiceImpl implements AuthService {
     private final TokenProvider tokenProvider;
 
     @Override
-    @Transactional(readOnly = true)
+    @Transactional
     public SignInResponse githubSignIn(GithubUserInfo githubUserInfo) {
-        log.info("githubUserInfo={}", githubUserInfo);
+        log.debug("githubUserInfo={}", githubUserInfo);
 
-        Optional<Member> member = memberRepository.findOneByGithubOauthId(githubUserInfo.getId());
+        Optional<Member> member = memberRepository.findByGithubOauthId(githubUserInfo.getId());
 
         if (member.isPresent()) {
+            String refreshToken = createRefreshToken(member.get());
+            Member updatedMember = member.get();
```

Comment:
순서를 바꾸면 더 좋지 않을까요? 큰 차이는 없을 것 같습니다.
```suggestion
            Member updatedMember = member.get();
            String refreshToken = createRefreshToken(updatedMember);
```